cluster/vmpreset-windows.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: kubevirt.io/v1alpha1
+kind: VirtualMachinePreset
+metadata:
+  name: windows-server-2012r2-preset
```

Comment:
I'd have done it the same way, but I'd omit the `-preset` suffix as it's redundant to the kind.